src/test/java/com/vishva/Payapp/Service/CreditServiceTest.java

```diff
@@ -64,14 +46,12 @@ public void accIdDoesNotExist() {
     }
 
     @Test
-    public void amountCredited() {
-        CreditRequest creditRequest = new CreditRequest("102", new BigDecimal("10"));
-        //AccountEntity accountEntity = new AccountEntity("123", "vishva", BigDecimal.ONE);
+    public void amountCreditedSuccessfully() {
+        CreditRequest creditRequest = new CreditRequest("134", new BigDecimal("105"));
+        AccountEntity accountEntity = new AccountEntity("134", "vishva", new BigDecimal("1500"));
+        doReturn(Optional.of(accountEntity)).when(creditRepositoryService).getAccountEntities(creditRequest.getAccId());
         creditService.amountCredited(creditRequest);
-        //doReturn(Optional.of(accountEntity)).when(accountRepository).findById(any());
-        //creditService.amountCredited(creditRequest);
-        //verify(accountRepository).save(any());
-        assertEquals("1010", accountRepository.findById(creditRequest.getAccId()).get().balance);
+        assertEquals(new BigDecimal("1605"), accountEntity.getBalance());
```

Comment:
This assertion is not valid. In this method only you are initializing the accountEntity.

I think no assertion is needed here as amountCredited method is returning nothing. You can only do verification here.